framework/src/executioners/FEProblemSolve.C

```diff
@@ -62,6 +62,7 @@ FEProblemSolve::validParams()
   params.addParam<unsigned int>("nl_max_funcs", 10000, "Max Nonlinear solver function evaluations");
   params.addParam<Real>("nl_abs_tol", 1.0e-50, "Nonlinear Absolute Tolerance");
   params.addParam<Real>("nl_rel_tol", 1.0e-8, "Nonlinear Relative Tolerance");
+  params.addParam<Real>("nl_div_tol", 1.0e+4, "Nonlinear Divergence Tolerance");
```

Comment:
It looks many bison tests are failing. This  may force us to skip this parameter by default.

Could you take a try: ` 1.0e+4` --> "-1"?

If we can pass all tests, then we should be able to merge it.